exist-core/src/main/java/org/exist/xmldb/AbstractRemoteResource.java

```diff
@@ -156,11 +154,7 @@ public void setLastModificationTime(final Date dateModified) throws XMLDBExcepti
             params.add(path.toString());
             params.add(dateModified.getTime());
 
-            try {
-                xmlRpcClientLease.get().execute("setLastModified", params);
-            } catch (final XmlRpcException e) {
-                throw new XMLDBException(ErrorCodes.UNKNOWN_ERROR, e.getMessage(), e);
-            }
+            collection.execute("setLastModified", params);
```

Comment:
should we make this string a (shared) constant?

Reply:
nope, there are too many of them?

Reply:
@dizzzz you mean `"setLastModified"`? Where should that be put to?

Reply:
yes. but forget about my remark as there are soo many, and... they are only used once right?

Reply:
yes